### ISSUE TITLE
go-ethereum/crypto prevents program from compiling successfully

### ISSUE THREAD

**Issue by i-fix-retarded-bugs** [ethereum/go-ethereum#30188]
#### System information

Irrelevant

#### Expected behaviour

Program compiles

#### Actual behaviour

```
github.com/ethereum/go-ethereum@v1.14.7/crypto/signature_nocgo.go:91:14: assignment mismatch: 2 variables but btc_ecdsa.SignCompact returns 1 value
```

#### Steps to reproduce the behaviour

Use `go-ethereum@v1.14.7` in your code and try to build your program.

P.S.
v.1.14.6 does not have this issue.

**Comment by i-fix-retarded-bugs** [comment]
@AlexanderMint

**Comment by fjl** [comment]
This has been fixed in https://github.com/ethereum/go-ethereum/pull/30181, which is not released yet. Please downgrade your btcec dependency version to fix this issue.

**Comment by AlexanderMint** [comment]
Before the new version is released, you can:

```go
 go get github.com/btcsuite/btcd/btcec/v2@v2.3.3
```

**Comment by echarrod** [comment]
Hi team 👋 any idea when this fix will be included in a release?

**Comment by fjl** [comment]
Fix is released now in v1.14.8.
